superdesk/commands/index_from_mongo.py

```diff
@@ -92,6 +94,11 @@ def get_mongo_items(cls, mongo_collection_name, page_size, last_id):
         db = app.data.get_mongo_collection(mongo_collection_name)
         args = {"limit": bucket_size, "sort": [(config.ID_FIELD, pymongo.ASCENDING)]}
 
+        try:
+            last_id = ObjectId(last_id)
```

Comment:
you should do it only if last_id is set because `ObjectId(None)` generates a valid ObjectId